src/components/dashboard/widget/dashboard-widget.component.ts

```diff
@@ -299,6 +319,8 @@ export class DashboardWidgetComponent implements OnInit, AfterViewInit, OnDestro
     static ngAcceptInputType_autoPositioning: boolean | string;
     static ngAcceptInputType_col: number | string;
     static ngAcceptInputType_row: number | string;
+    static ngAcceptInputType_minColSpan: number | string;
+    static ngAcceptInputType_minRowSpan: number | string;
```

Comment:
We can use the type `NumberInput` from the CDK Coercion library as the type for this. We can also now change the others here, there is also `BooleanInput` as well. We couldn't previously use those because they were only in CDK v9+ but now we don't support Angular 8 we should now be using these provided types.